utils/db/repos.ts

```diff
@@ -3,6 +3,36 @@ import conn from '.';
 import type { DbRepo, RepoIdentifier } from '../../types/repository';
 import { convert } from './converter';
 
+export const saveRepoIdentifierToDb = async (repos: RepoIdentifier[], topicId: string) => {
+	const insertRepoQuery = `
+		INSERT INTO repos (
+			repo_provider,
+			repo_owner,
+			repo_name,
+			install_id,
+			is_installed,
+			created_at
+		) VALUES
+		${repos.map(repo => `(
+			${convert(repo.repo_provider)},
+			${convert(repo.repo_owner)},
+			${convert(repo.repo_name)},
+			${convert(topicId)},
+			${convert(false)},
```

Comment:
something seams wrong here. This should be an array of strings.

Reply:
redundant